azure/src/createAzureClient.ts

```diff
@@ -140,7 +140,7 @@ export class CorrelationIdPolicy implements PipelinePolicy {
 
     public async sendRequest(request: PipelineRequest, next: SendRequest): Promise<PipelineResponse> {
         const headerName = 'x-ms-correlation-request-id';
-        const id: string = this.context.telemetry.properties[headerName] ||= uuidv4();
+        const id: string = (this.context.telemetry.properties[headerName] as string | undefined) ||= uuidv4();
```

Comment:
@alexweininger Just checking, but I was getting a type error here due to the new possibility for TrustedTelemetry type, what's the recommended way to handle this, do we need to typecast?

Reply:
I say that we don't allow generic telemetry properties be TrustedTelemetryValue types. We should reserve that for only specific properties that we sign off on as trusted. Otherwise we have to deal with this annoying typing and I find that it's a bit of a security/privacy risk,